src/middlewared/middlewared/etc_files/local/collectd.conf

```diff
@@ -0,0 +1,281 @@
+<%
+	import os
```

Comment:
All this complicated logic does not belong to `.conf` template. Please move it to some private middleware method. With current approach you (probably) don't have syntax highlight, linting or hinting, you can't test this code, travis is also not able to check it.